src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { colorMap, size, fontSize } from './theme';
+import { P } from './text';
+
+const Tab = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const activeStyles = css`
+    color: ${colorMap.devWhite};
+    border-bottom-color: ${colorMap.darkGreen};
+    &:hover {
+        color: ${colorMap.devWhite};
+    }
+`;
+
+const defaultStyles = css`
+    &:hover {
+        color: ${colorMap.devWhite};
+    }
+`;
+
+const TabButton = styled('button')`
+    border: none;
+    outline: none;
+    cursor: pointer;
+    transition: 0.3s;
+    background-color: inherit;
+    width: 136px;
+    padding: ${size.micro} ${size.medium} ${size.mediumLarge}
+        ${size.mediumLarge};
+    border-bottom: 3px solid transparent;
+
+    font-size: ${fontSize.default};
+    color: ${colorMap.greyDarkOne};
+    font-family: 'Fira Mono';
```

Comment:
Since `P` works now are these lines all still needed? Color might be but I suspect font-family and size are not

Reply:
It looks like we do still need them. `P` doesn't specify a font-family and the size is smaller than the one specified in the design.

Reply:
Ah okay! Thanks for digging into this